Lib/test/test_zlib.py

```diff
@@ -44,10 +44,7 @@
 #   zlib.decompress(func1(data)) == zlib.decompress(func2(data)) == data
 #
 # Make the assumption that s390x always has an accelerator to simplify the skip
-# condition. Windows doesn't have os.uname() but it doesn't support s390x.
-skip_on_s390x = unittest.skipIf(hasattr(os, 'uname') and os.uname().machine == 's390x',
-                                'skipped on s390x')
-
```

Comment:
For reference: this moved `skip_on_s390x` away from its comment that explains it. #125041 fixes that.